install/checkmk-install.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+#Copyright (c) 2021-2024 community-scripts ORG
+# Author: Michel Roegl-Brunner (michelroegl-brunner)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  curl \
+  mc 
+msg_ok "Installed Dependencies"
+
+msg_info "Install Checkmk"
+RELEASE=$(curl -fsSL https://api.github.com/repos/checkmk/checkmk/tags | grep "name" | awk '{print substr($2, 3, length($2)-4) }' | grep -v "*-rc" | tail -n +2 | head -n 1)
+wget -q https://download.checkmk.com/checkmk/${RELEASE}/check-mk-raw-${RELEASE}_0.bookworm_amd64.deb -O /opt/checkmk.deb
+$STD apt-get install -y /opt/checkmk.deb
```

Comment:
apt install or dpkg -i?

Reply:
The guid says apt install, nut i can change it if you like.

![cmk](https://github.com/user-attachments/assets/4045cef4-b1d3-4a43-9c69-09dedd9fb675)

Reply:
if it works, try it

Reply:
Changed and it is working

Reply:
can you execute: apt-get -f install after dpkg -i ? (not in the script, in the finished lxc). If something installed, we need apt-get install ^^

Reply:
got nothing.
![image](https://github.com/user-attachments/assets/71773ff3-d45b-42a7-98b5-1f80920552ce)